Prog/HMDA_sum_P09.sas

```diff
@@ -0,0 +1,174 @@
+/**************************************************************************
+ Program:  HMDA_sum_P09.sas
+ Library:  HMDA
+ Project:  NeighborhoodInfo DC
+ Author:   Rob Pitingolo
+ Created:  9/11/18
+ Version:  SAS 9.4
+ Environment:  Windows 7
+ 
+ Description:  Create HMDA summary indicator file for Census tracts
+ (2000) from selected variables in HMDA files.
+
+ When updating: Make sure there is an updated version of %hud_inc_yyyy
+			    in the iPums macro library. 
+ 
+ Modifications:
+ 
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas"; 
+
+** Define libraries **;
+%DCData_lib( HMDA )
+%DCData_lib( ipums )
+
+%let start_yr = 2009;
+%let end_yr = 2016;
+
+%let revisions = New file ;
+
+%let tr2000data = hmda2009tr hmda2010tr hmda2011tr;
+%let tr2010data = hmda2012tr hmda2013tr hmda2014tr hmda2015tr hmda2016tr;
+
+
+
+/*Create tract-level summary file for each year of data */
+%hmda_tracts_new (&start_yr.,&end_yr.);
+
+
+/* Merge 2000-tract year files */
+data hmda_long00_tr00;
+	merge &tr2000data.;
+	by geo2000;
+run;
+
+proc contents data = hmda_long00_tr00 out = vars2000;
+run;
+
+data vars2000p;
+	set vars2000;
+	prefix = upcase(substr(name,1,3));
+run;
```

Comment:
Loan count variables with missing values should be recoded to 0, since they represent no lending activity.